projects/js-packages/components/components/alert/index.tsx

```diff
@@ -19,9 +18,21 @@ export const LEVEL_WARNING = 'warning';
 export const LEVEL_INFO = 'info';
 export const LEVEL_SUCCESS = 'success';
 
-export const ALERT_LEVELS = [ LEVEL_ERROR, LEVEL_WARNING, LEVEL_INFO, LEVEL_SUCCESS ];
+export const ALERT_LEVELS = [ LEVEL_ERROR, LEVEL_WARNING, LEVEL_INFO, LEVEL_SUCCESS ] as const;
+export type alertLevelProp = typeof ALERT_LEVELS[ number ];
```

Comment:
So, after our discussion, this needs to be updated, right?

Reply:
Not sure actually. I left a comment raising up about the benefits about to keep using the current implementation  [here](https://github.com/Automattic/jetpack/pull/24204#discussion_r863069494)

Reply:
let's continue in the thread mentioned above